google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
Yes, it is intended usage to import a separate LRO client. That said, in Java, the plan is to provide an instance of that LRO client through an accessor on the speech client.